openxr/examples/fullscreen.vert

```diff
@@ -1,8 +1,27 @@
 #version 450
+#extension GL_EXT_multiview : require
+
+layout(location = 0) in vec3 vPosition;
 
 layout(location = 0) out vec2 screen_coords;
+layout(location = 1) out vec3 vColor;
+
+layout(push_constant) uniform ModelTransform {
+    mat4 model;
+};
+
+struct Transform {
+    mat4 view;
+    mat4 proj;
```

Comment:
If we don't need these to be separate, they should be multiplied together in advance on the CPU.